json-endpoints.php

```diff
@@ -3388,3 +3389,32 @@
 		'headers' => array( 'authorization' => 'Bearer YOUR_API_TOKEN' ),
 	),
 ) );
+
+new WPCOM_JSON_API_Bulk_Delete_Post_Endpoint( array(
+	'description' => 'Delete multiple posts. Note: If the trash is enabled, this request will send non-trashed posts to the trash. Trashed posts will be permanently deleted.',
+	'group'       => 'posts',
+	'stat'        => 'posts:1:bulk-delete',
+	'min_version' => '1.1',
+	'max_version' => '1.1',
+	'method'      => 'POST',
+	'path'        => '/sites/%s/posts/delete',
+	'path_labels' => array(
+		'$site'    => '(int|string) Site ID or domain',
+	),
+	'request_format' => array(
+		'post_ids' => '(array) An array of Post IDs to delete or trash.',
+	),
+
```

Comment:
I think we also need to specify the `response_format`, both here and sync on `wpcom` side. Otherwise, any response coming from Jetpack site will be filtered out.